tests/testthat/test-build-home-community.R

```diff
@@ -0,0 +1,34 @@
+context("test-build-home-community.R")
+
+test_that("has_coc works", {
+  skip_if_not(dir_exists(path(test_path("assets/site-dot-github"),
```

Comment:
Why wouldn't this exist?

Reply:
I saw https://github.com/r-lib/pkgdown/blob/56fe09351537ee9860d93461aa22f720650a3377/tests/testthat/test-build_home.R#L54

But maybe I made a wrong interpretation?

Reply:
I got failures on Travis prior to seeing and copying that line.

Reply:
I see https://github.com/r-lib/pkgdown/pull/706.

Reply:
You just need to copy the comment too, so the reasoning is preserve.